inst/extdata/database/tedfs/Tech/Hydrogen Storage.csv

```diff
@@ -1,9 +1,9 @@
 subtech,variable,reference_variable,period,value,uncertainty,unit,reference_value,reference_unit,comment,source,source_detail
-AGCS,Total Production Expenditure,Input|Hydrogen,,0.33,,USD,1.0,kg,,Abdin21,
-AGCS,Total Production Expenditure,Input|Hydrogen,,25.2,,USD,1.0,kg,,Abdin21,
-SC,Total Production Expenditure,Input|Hydrogen,,0.14,,USD,1.0,kg,,Abdin21,
-AGCS,CAPEX,Storage Capacity|h2,2017.0,100.0,,EUR,1.0,m³;norm,"here, H2 is stored at 500 bar. ""under standard conditions, (atmospheric pressure and 0°C), at 500 bar, H2 has a density of 33kg/m^3"" ",Gorre19,
-AGCS,OPEX Fixed Relative,,2017.0,1.5,,percent,,,,Gorre19,
-AGCS,CAPEX,Storage Capacity|h2,2030.0,75.0,,EUR,1.0,m³;norm,"here, H2 is stored at 500 bar. ""under standard conditions, (atmospheric pressure and 0°C), at 500 bar, H2 has a density of 33kg/m^3"" ",Gorre19,
-AGCS,OPEX Fixed Relative,,2030.0,1.5,,percent,,,,Gorre19,
-SC,Total Production Expenditure,Input|Hydrogen,2020.0,0.24,,EUR,1.0,kWh,40 days average storage duration,Michalski17,
+AGCS,Total Production Expenditure,Storage Capacity|Hydrogen,,0.33,,USD,1.0,kg,"Data point should be checked, it couldn't be found in the corresponding source paper.",Abdin21,
+AGCS,Total Production Expenditure,Storage Capacity|Hydrogen,,25.2,,USD,1.0,kg,"Data point should be checked, it couldn't be found in the corresponding source paper.",Abdin21,
+SC,Total Production Expenditure,Storage Capacity|Hydrogen,,0.14,,USD,1.0,kg,"Data point should be checked, it couldn't be found in the corresponding source paper.",Abdin21,
```

Comment:
I found the reason behind the confusion - the wrong paper was being cited in the bibtex file.

Reply:
> On another note, I'm wondering about what variables really make sense to report in the case of storage technologies. `Total Production Expenditure` seems a bit off given that nothing is actually produced.

What would you suggest ? These values are levelized cost of hydrogen storage in the end. Should they be reported like that, or should they be removed from the TEDs altogether ?

Reply:
Well done on spotting the incorrect citation.

And yes, I'm thinking that perhaps we should simply remove entries that report "levelized cost of hydrogen storage" as you say. Because we're trying to gather technology data here, whereas that number would be highly dependent on application and context.